modules/nested-tabs/module.php

```diff
@@ -26,5 +26,21 @@ public function __construct() {
 				'nested-elements',
 			], ELEMENTOR_VERSION, true );
 		} );
+
+		add_action( 'elementor/frontend/after_register_styles', [ $this, 'register_style' ] );
+	}
+
+	/**
+	 * Register styles.
+	 *
+	 * @return void
+	 */
+	public function register_style() {
+		wp_register_style(
+			$this->get_name(),
+			$this->get_css_assets_url( 'frontend', 'assets/css/modules/nested-tabs/' ),
+			[],
```

Comment:
@rami-elementor - Are you sure that no-dependency is needed? I see that there is a base-tabs.scss file for example.